sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Configurations that allow callers to specify details about how to execute
+    /// an Extractive Text Summarization action in a set of documents.
+    /// For example, set the model version, specify the order in which extracted
+    /// sentences are returned, and more.
+    /// </summary>
+    public class ExtractSummaryAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
+        /// class which allows callers to specify details about how to execute
+        /// an extractive text summarization action in a set of documents.
+        /// For example, set the model version, specify the order in which extracted
+        /// sentences are returned, and more.
+        /// </summary>
+        public ExtractSummaryAction()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
+        /// class based on the values of a <see cref="ExtractSummaryOptions"/>.
+        /// It sets the <see cref="ModelVersion"/>, <see cref="DisableServiceLogs"/>,
+        /// <see cref="MaxSentenceCount"/>, and <see cref="OrderBy"/> properties.
+        /// </summary>
+        public ExtractSummaryAction(ExtractSummaryOptions options)
+        {
+            ModelVersion = options.ModelVersion;
+            DisableServiceLogs = options.DisableServiceLogs;
+            MaxSentenceCount = options.MaxSentenceCount;
+            OrderBy = options.OrderBy;
+        }
+
+        /// <summary>
+        /// Gets or sets a value that, if set, indicates the version of the text
+        /// analytics model that will be used to generate the result. For supported
+        /// model versions, see operation-specific documentation, for example:
+        /// <see href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/model-versioning#available-versions"/>.
+        /// </summary>
+        public string ModelVersion { get; set; }
+
+        /// <summary>
+        /// The default value of this property is 'false'. This means, the Language service logs your input text for 48 hours,
```

Comment:
Nit: If you're going to copy/paste or retype the summary, I'd put programmatic elements like `'false'` like so `<c>false</c>` to avoid looking like a string.